nvflare/private/fed/client/communicator.py

```diff
@@ -195,25 +195,26 @@ def pull_task(self, servers, project_name, token, ssid, fl_ctx: FLContext):
             size = len(task.payload)
             task.payload = fobs.loads(task.payload)
             task_name = task.payload.get_header(ServerCommandKey.TASK_NAME)
+            task.payload.add_cookie(ReservedHeaderKey.SSID, ssid)
             if task_name not in [SpecialTaskName.END_RUN, SpecialTaskName.TRY_AGAIN]:
                 self.logger.info(
                     f"Received from {project_name} server "
-                    f" ({size} Bytes). getTask: {task_name} time: {end_time - start_time} seconds"
+                    f" ({size} Bytes). pull_task: {task_name} time: {end_time - start_time} seconds"
```

Comment:
Let's not change these words - people have been looking for these words in their work.

Reply:
thanks, fixed.